arwen/elrondapi/elrondei.go

```diff
@@ -1942,17 +2068,25 @@ func v1_4_callValue(context unsafe.Pointer, resultOffset int32) int32 {
 
 //export v1_4_getESDTValue
 func v1_4_getESDTValue(context unsafe.Pointer, resultOffset int32) int32 {
-	_ = failIfMoreThanOneESDTTransfer(context)
+	runtime := arwen.GetRuntimeContext(context)
+	runtime.SetInitialGasInGasTrace(getESDTValueName)
```

Comment:
delete set initial gas in gas trace function

Reply:
so then if you call `getESDTValue()` your gas would be traced under `getESDTValueByIndex`, it would have been more specific, but I will change

Reply:
will do the same for the rest of pair functions like `functionName`, `functionNameByIndex`

Reply:
that is fair